modules/xfem/test/tests/bimaterials/glued_bimaterials_2d.i

```diff
@@ -0,0 +1,298 @@
+[GlobalParams]
```

Comment:
Would you add a few sentences at the top of each input file about the problem set up and the expected solution please?

Reply:
I agree. These input files need short descriptions of what they are expected to do at the top.

Reply:
done!